test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml

```diff
@@ -0,0 +1,202 @@
+---
+kind: Template
```

Comment:
Do we want to commit templates to our testing repo, as opposed to download them with every prod build?

Reply:
Ok, thanks for explanation.

Reply:
And "signle" template is used if there isn't a prod one.